.jules/friction/open/compat-wasm-pack-args.md

```
@@ -0,0 +1,21 @@
+# Friction Item
+
+id: compat-wasm-pack-args
+persona: compat
+style: builder
+shard: bindings-targets
+status: open
+
+## Problem
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)